stock_putaway_product/__init__.py

```diff
@@ -0,0 +1,5 @@
+# © 2016 Jos De Graeve - Apertoso N.V. <Jos.DeGraeve@apertoso.be>
```

Comment:
© -> Copyright
adding
`# Copyright 2018 Camptocamp SA
`
would be a good move ;)
same for other files as well